packages/color-wheel/README.md

```diff
@@ -0,0 +1,63 @@
+## Description
+
+An `<sp-color-wheel>` lets users visually change an individual channel of a color on a circular track.
+
+### Usage
+
+[![See it on NPM!](https://img.shields.io/npm/v/@spectrum-web-components/color-wheel?style=for-the-badge)](https://www.npmjs.com/package/@spectrum-web-components/color-wheel)
+[![How big is this package in your project?](https://img.shields.io/bundlephobia/minzip/@spectrum-web-components/color-wheel?style=for-the-badge)](https://bundlephobia.com/result?p=@spectrum-web-components/color-wheel)
+
+```
+yarn add @spectrum-web-components/color-wheel
+```
+
+Import the side effectful registration of `<sp-color-wheel>` via:
+
+```
+import '@spectrum-web-components/color-wheel/sp-color-wheel.js';
+```
+
+When looking to leverage the `ColorWheel` base class as a type and/or for extension purposes, do so via:
+
+```
+import { ColorWheel } from '@spectrum-web-components/color-wheel';
+```
+
+## Color Formatting
+
+When using the color elements, use `el.color` to access the `color` property, which should manage itself in the colour format supplied. If you supply a color in `rgb()` format, `el.color` should return the color in `rgb()` format, as well.
+
+The current color formats supported are as follows:
+
+-   Hex3, Hex4, Hex6, Hex8
+-   HSV, HSVA
+-   HSL, HSLA
+-   RGB, RGBA
+-   Strings (eg "red", "blue")
+
+**Please note for the following formats: HSV, HSVA, HSL, HSLA**
+When setting a color's lightness or value to 100%, the hue and saturation value are not preserved. This is detailed in the [TinyColor documentation](https://www.npmjs.com/package/@ctrl/tinycolor). Currently, the Spectrum Web Components has a workaround to support the preservation of the hue.
```

Comment:
Is it lightness and value at 100% or is it one at 100% and the other at 0%?

Can we say may not be preserved instead, and connect it directly to the color propertY of the element?

The `value` property is manages with the intent to be true to the `hue` regardless of the `color` from which is it originally derived.